src/test/java/io/sniffy/socket/TcpExpectationTest.java

```diff
@@ -0,0 +1,36 @@
+package io.sniffy.socket;
+
+import io.sniffy.Sniffer;
+import io.sniffy.Spy;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+
+public class TcpExpectationTest extends BaseSocketTest {
+
+    @Before
+    public void installSocketFactory() throws IOException {
+        SnifferSocketImplFactory.install();
+    }
+
+    @After
+    public void uninstallSocketFactory() {
+        SnifferSocketImplFactory.uninstall();
+    }
+
+    @Test
+    public void testExactConnections() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/sniffy/sniffy/file/2496468357/issues/source?bid=3005960&fileBranchId=3034483#l24)